docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -307,6 +307,27 @@ max-peers=42
 Specifies the maximum P2P peer connections that can be established.
 The default is 25.
 
+### max-remote-connections-percentage
+
+```bash tab="Syntax"
+--max-remote-connections-percentage=<DOUBLE>
+```
+
+```bash tab="Command Line"
+--max-remote-connections-percentage=25
+```
+
+```bash tab="Environment Variable"
+PANTHEON_MAX_REMOTE_CONNECTIONS_PERCENTAGE=25
+```
+
+```bash tab="Configuration File"
+max-remote-connections-percentage=25
+```
+
+Percentage of remote wire connections that can be established with the node. Must be between 0 and 100 inclusive.
```

Comment:
If we want to be consistent with the CLI help, that text is currently:
```suggestion
The maximum percentage of P2P connections that can be initiated remotely. Must be between 0 and 100 inclusive.
```

Reply:
The CLI doc isn't consistent with CLI help